media-gfx/hugin/hugin-2016.2.0.ebuild

```diff
@@ -18,7 +18,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~ppc ~x86"
 
 LANGS=" ca ca-valencia cs da de en-GB es eu fi fr hu it ja nl pl pt-BR ro ru sk sv zh-CN zh-TW"
-IUSE="debug lapack python sift $(echo ${LANGS//\ /\ l10n_})"
+IUSE="debug +doc lapack python sift $(echo ${LANGS//\ /\ l10n_})"
```

Comment:
do you really want to make building docs the default? This is afaik not a general Gentoo default.